src/SarifWeb/Views/Home/Index.cshtml

```diff
@@ -235,7 +238,8 @@
                 <img id="screenshot01" src="~/Images/Screenshots/VSViewer.thumbnail.png" index="0" coloration="dark" role="button" aria-labelledby="vsViewerCardTitle" alt="Visual Studio extension" width="360" height="240" />
             </div>
             <div class="viewer-buttons">
-                <button role="button" aria-label="Install VS Viewer extension" href="https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer" target="_blank" class="sarifweb-button">Install</button>
+                <button role="button" aria-label="Install VS Viewer extension" href="https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer" target="_blank" class="sarifweb-button">Install 2019</button>
+                <button role="button" aria-label="Install VS Viewer extension" href="https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer2022" target="_blank" class="sarifweb-button">Install 2022</button>
```

Comment:
@EasyRhinoMSFT follow up on our offline conversation. How does this look?

Reply:
Anything longer than `Install 2019` don't fit on the button.

Reply:
This is exactly what I was thinking :thumbsup: